index.js

```diff
@@ -125,7 +162,12 @@ AssetsWebpackPlugin.prototype = {
               if (type === 'string') {
                 typeMap[typeName] = [typeMap[typeName]]
               }
-              typeMap[typeName].push(combinedPath)
+              if(self.options.includeLazyChildAssets && loadingBehaviour){
+                typeMap[typeName + ':' + loadingBehaviour] = typeMap[typeName + ':' + loadingBehaviour] || [];
+                typeMap[typeName + ':' + loadingBehaviour].push(combinedPath) 
```

Comment:
Think these lines could be cleaned up by extracting out the name:behavior to a const.

ex) 
```js
	const lazyAsset = typeName + ':' + loadingBehaviour;
	typeMap[lazyAsset] = typeMap[lazyAsset] || [];
	typeMap[lazyAsset].push(combinedPath)
```